### PR TITLE
SERVER-87143 Skip WouldChangeOwningShardBatchWrite workload on versions < 6.0

### PR DESCRIPTION
**Pull request by toto-dev**
**Jira Ticket:** [SERVER-87143](https://jira.mongodb.org/browse/SERVER-87143)

### Whats Changed

Skip  WouldChangeOwningShardBatchWrite workload on versions < 6.0.

[🌲 EVG](https://spruce.mongodb.com/version/66961fda4521620007b5f050/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)

### CODE REVIEW

File: src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml [hunk 1]
```diff
@@ -93,4 +93,5 @@ AutoRun:
           - shard-lite-80-feature-flags
       branch_name:
         $neq:
-          - v4.0
+          - v4.4
```
Comment (afuschetto) [inline src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml:96]: Do we run this on the 4.4 branch? Why?

Comment (toto-dev) [reply src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml:96]: @afuschetto we do still have [4.4 branch for sys-perf project in Evergreen CI](https://spruce.mongodb.com/commits/sys-perf-4.4). The last executed patch on this branch was on May 7th, 2024.

Not sure why we still run 4.4 system performance patches

File: src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml [hunk 2]
```diff
@@ -93,4 +93,5 @@ AutoRun:
           - shard-lite-80-feature-flags
       branch_name:
         $neq:
```
Comment (Fefer-Ivan) [inline src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml:95]: Instead of listing versions, you can use $gte: v6.0

Comment (toto-dev) [reply src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml]: Done

**Comment by toto-dev** [comment]
[🌲 EVG](https://spruce.mongodb.com/version/6698fdf042c0920007ff320f/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)

**Review by Fefer-Ivan — APPROVED** [review]

**Pull request merged**